src/Modules/ScoreboardModule/Services/ScoreboardService.cs

```diff
@@ -29,6 +31,8 @@ public async Task SendScoreboardAsync()
     {
         var currentNextMaxPlayers = await server.Remote.GetMaxPlayersAsync();
         var currentNextMaxSpectators = await server.Remote.GetMaxSpectatorsAsync();
+
+        dynamic theme = themeManager.Theme;
```

Comment:
This doesn't belong here, does it? I can't see where it's used.
Or is it needed to initialize it for that module?